haystack/fields.py

```diff
@@ -134,7 +135,10 @@ def prepare_template(self, obj):
             template_names = ['search/indexes/%s/%s_%s.txt' % (app_label, model_name, self.instance_name)]
 
         t = loader.select_template(template_names)
-        return t.render(Context({'object': obj}))
+        if django.VERSION >= (1, 8, 0):
```

Comment:
This check can now be removed since we no longer support Django 1.7 or earlier now that the Django core team has dropped support